u_parrots.go

```diff
@@ -624,7 +664,23 @@ func removeRandomCiphers(s []uint16, maxRemovalProbability float32) []uint16 {
 			i--
 		}
 	}
-	return s
+	return s[:sliceLen]
+}
+
+func removeRC4Ciphers(s []uint16) []uint16 {
+	// removes elements in place
+	sliceLen := len(s)
+	for i := 0; i < sliceLen; i++ {
+		cipher := s[i]
+		if cipher == TLS_ECDHE_ECDSA_WITH_RC4_128_SHA ||
+			cipher == TLS_ECDHE_RSA_WITH_RC4_128_SHA ||
+			cipher == TLS_RSA_WITH_RC4_128_SHA {
```

Comment:
Aren't there other RC4 ciphers? e.g. https://tlsfingerprint.io/find/cipher/c002
Might want to at least comment this assumes only a subset of all RC4 ciphers (there's a bunch of weak ones that presumably wouldn't ever get to this function, but it is named removeRC4Ciphers, not removeLikelyRC4: https://tlsfingerprint.io/weak-ciphers )

Reply:
those are the only RC4 ciphers implemented by Golang and returned by `defaultCipherSuites()`